composer_class.rb

```diff
@@ -0,0 +1,11 @@
+class Composer
+  attr_accessor :first_name, :last_name, :year_born, :year_died, :age
```

Comment:
For variable that aren‘t changed from outside the object, use `attr_reader` instead. I think it is the case here so use:

```suggestion
  attr_reader :first_name, :last_name, :year_born, :year_died, :age
```